storage/sealer/commitment/commr.go

```diff
@@ -25,7 +25,7 @@ func CommR(commC, commRLast [32]byte) ([32]byte, error) {
 	input_b.SetBytes(commRLast[:])
 	input := []*big.Int{input_a, input_b}
 
-	cons, err := poseidon.GenPoseidonConstants(3)
+	cons, err := poseidon.GenPoseidonConstants[*fr.Element](3)
```

Comment:
Confirming that there _is_ a golden test for this here: https://github.com/filecoin-project/lotus/blob/f6d4fc7ccb0a53fd49cf01a82ab3a9528345afab/storage/sealer/commitment/commr_test.go#L9-L35.

Reply:
Correct, that test is exactly that